src/i18n/ru/ui.ts

```diff
@@ -133,5 +133,5 @@ export default UIDictionary({
 	'starlight.title': 'Хотите создать свою собственную документацию?',
 	'starlight.description': 'Возьмите этот шаблон, чтобы начать.',
 	// `<StudioHeading>` component
-	'studioHeading.label': 'Особенность Studio',
+	'studioHeading.label': 'Функция Studio',
```

Comment:
"Особенность Studio" would be more appropriate, not "Функция". This aligns with the notion of introducing a distinct feature/characteristic of Studio.
```suggestion
	'studioHeading.label': 'Особенность Studio',
```